README.md

```diff
@@ -5,16 +5,34 @@
 
 ```bash
 # development
+$ git checkout develop
 $ cp .env.dist .env
 $ cp docker-compose.dev.yml docker.compose.yml
```

Comment:
should'nt it be `cp docker-compose.dev.yml docker-compose.yml` here ? ( '-' instead of '.')

Reply:
yes, will make this change, thanks